assets/scss/_gift-config.scss

```diff
@@ -129,26 +131,25 @@ div[data-toggle="buttons"] label:active, div[data-toggle="buttons"] label.active
 }
 
 label.custom-amount {
-  border: none !important;
-  padding-left: 0px !important;
+  //border: none !important;
+  //padding-left: 0px !important;
+  // background: none;
+  // width: 125px;
 }
 
 label.btn.btn-default-form.custom-amount.mr.u-textLeft {
   border: none;
   padding-left: 0px;
 }
 
-label.btn.btn-default-form {
+/* label.btn.btn-default-form {
```

Comment:
If it's not in use, then just delete this class too. Double check the app for its use though before deleting. Probably safe, but double check.